services/frontend/src/components/SignIn.vue

```diff
@@ -37,8 +41,37 @@
 </template>
 
 <script>
+import router from '../router'
+
 export default {
-  name: 'Login'
+  name: 'Login',
+  data() {
+    return {
+      username: null,
+      password: null
+    }
+  },
+  methods: {
+    checkLogin() {
+      // TODO: this is a mock
+      if (this.username == "test" && this.password == "test") {
+        this.acceptLogin();
+      }
+      else {
+        this.rejectLogin();
+      }
+    },
+    acceptLogin() {
+      console.log('Logowanie powiodło się');
+      router.push('home')
+    },
+    rejectLogin() {
+      this.username=null;
+      this.password=null;
+      console.log('Logowanie nie powiodło się');
+      alert("Niepoprawny login");
```

Comment:
Do plan using alerts in the final version? Personally alerts just annoy me, i'd add notification system just to get rid of alerts, furthermore we  could use such system to give notifications then analysis has finished. Vue should have lib that will take care of that, and vuetify has nice notifications if we need them

Reply:
no i don't plan on that, but we need to show on monday that app will have some sort of feedback in that case, but got no time for something more elegant right now

Reply:
agree for now this will do